src/pages/fieldManagement/settings/defaultFieldComponents/Editors.jsx

```diff
@@ -3,40 +3,38 @@ import { get } from 'lodash-es';
 
 import TreeEditor from './TreeEditor';
 import ConfigureDefaultField from './ConfigureDefaultField';
+import RelationshipEditor from './RelationshipEditor';
 
-export function RelationshipEditor() {
-  return null;
-}
-
-// export function RelationshipEditor({
-//   onClose,
-//   onSubmit,
-//   formSettings,
-//   setFormSettings,
-//   ...rest
-// }) {
-//   const speciesOptions = get(formSettings, 'relationships', []);
+export function RelationshipEditorWrapper({
+  onClose,
+  onSubmit,
+  formSettings,
+  setFormSettings,
+  children,
+  ...rest
+}) {
+  const transformedRelationshipOptions = get(
+    formSettings,
+    'relationships',
+    formSettings,
+  );
```

Comment:
this seems weird. maybe just double-check?

Reply:
The reason for this is that formSettings initially looks like {relationships:{}, species:{}, regions{}}, but in subsequent renderings is just the relationships subset. I think that it's totally ok that it loses species and region key-value pairs (do we call them elements or attributes for objects??), because the user cannot simultaneously edit all three anyway (i.e., the save button in the dialog ensures this). 
Given that the formSettings can enter that component one of two ways, this was how I solved that issue.
I tried dumber ways as well ⚰️ .

Reply:
this seems really weird! is that how i had it set it up for species and regions as well? sounds like a mistake to me

Reply:
I'm guessing that this has something to do with all of them belonging to the same form schema. IMHO they don't need to be.

Reply:
In your onChange in defined in the RelationshipEditor, you are just setting the form state to relationships, which explains the behavior you are seeing. You need to include the entire form state. I think that all relationships are provided to the onChange, so the following should work:
```
onChange={relationships => {
    setFormSettings({ 
        ...formSettings,
        relationships
      });
  }}

```